### ISSUE TITLE
Windows doen't have the attribute "SIGALRM" for the module signal

### ISSUE THREAD

**Issue by briandrawert** [GillesPy2/GillesPy2#307]
```
Traceback (most recent call last):

   File "C:\Users\noviello\.spyder-py3\temp.py", line 35, in <module>
     results = model.run(number_of_trajectories=10)

   File
"C:\Users\noviello\AppData\Roaming\Python\Python37\site-packages\gillespy2\core\gillespy2.py",
line 634, in run
     with time_out(timeout):

   File "C:\Users\noviello\Anaconda3\lib\contextlib.py", line 112, in
__enter__
     return next(self.gen)

   File
"C:\Users\noviello\AppData\Roaming\Python\Python37\site-packages\gillespy2\core\gillespy2.py",
line 610, in time_out
     signal.signal(signal.SIGALRM, raise_time_out)
```
I have the 1.3.1 release of Gillespy2 on Python 3.7.

**Comment by briandrawert** [comment]
Duplicate of #300 
Fixed by #301

**Comment by briandrawert** [comment]
Possibly fixed by #304

**Comment by briandrawert** [comment]
fixed with #304
